### PR TITLE
manifest: Add static sysusers entries

### PR DESCRIPTION
**Pull request by cgwalters**
For use with https://github.com/projectatomic/rpm-ostree/pull/1679

### CODE REVIEW

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -7,6 +7,16 @@ tmp-is-dir: true
 
 # Required by Ignition, and makes the system not compatible with Anaconda
 machineid-compat: false
+# New model
+sysusers: true
```
Comment (jlebon) [inline fedora-coreos-base.yaml]: Hmm, how this intersects with Ignition is interesting. Ignition allows you to specify users/groups by name for each file/dir/link. I think this means that we'd have to run sysusers too in the `initrd` on first boot. That would also allow us to just do a blanket `systemd-tmpfiles --create --boot --root=/sysroot --prefix=/var`.

Comment (cgwalters) [reply fedora-coreos-base.yaml]: Yep see https://github.com/coreos/fedora-coreos-tracker/issues/155

Comment (cgwalters) [reply fedora-coreos-base.yaml]: Where's the code that runs `systemd-tmpfiles` for Ignition today?

Comment (jlebon) [reply fedora-coreos-base.yaml]: Today, nowhere. :)
Soon: https://github.com/coreos/ignition-dracut/pull/47

**Comment by dustymabe** [comment]
@jlebon switched over the default branch to `testing-devel` and deleted `master`. This action closed off all the pending PRs against master. If still relevant can you re-open and edit your PR to target `testing-devel`?

**Pull request closed without merging**